paypalscript.py

```diff
@@ -27,6 +29,16 @@ def read(self):
         except FileNotFoundError:
             exit(f"File { self.in_file } not found")
 
+    def validate(self, validation_date):
+        validated_data = []
+        for row in self.data:
+            date_time_obj = datetime.strptime(row["Date"] + " " + row["Time"], "%m/%d/%Y %H:%M:%S")
```

Comment:
Use an f-string here and try not to mix single and double quotes unless using quotes in a string:

date_time_obj = datetime.strptime(f '{ row['Date'] }  { row['Time'] }", "%m/%d/%Y %H:%M:%S")